Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,12 +1,264 @@
 ﻿using System;
+using System.Text;
+using System.Threading;
 
 namespace BootCamp.Chapter
 {
-    class Program
+    /// <summary>
+    /// I decided to treat this like a very small game and created
+    /// a menu for the requirements I had to cover.
+    /// </summary>
+    public static class Program
     {
-        static void Main(string[] args)
+        /// <summary>
+        /// In introduced Thread.Sleep() to help with program functionality
+        /// </summary>
+        private const int displayTimeout = 3000;
+
+        // stored Unicode character codes in constants
+        private const char arrowUp = '\u21a5';
+
+        private const char arrowDown = '\u21a7';
+        private const char arrowLeft = '\u21a4';
+        private const char arrowRight = '\u21a6';
+
+        private static void Main(string[] args)
+        {
+            Console.OutputEncoding = Encoding.UTF8;
+            MainMenu();
+        }
+
+        private static void MainMenu()
+        {
+            // hide typing cursor and clear previous screen
+            Console.Clear();
+            Console.CursorVisible = false;
+            Console.WriteLine("Main Menu");
+            Console.WriteLine("  (1) Conversion =>");
+            Console.WriteLine("  (2) Play with WASD arrows.");
+            Console.WriteLine("(Esc) Quit!");
+
+            while (true)
```

Comment:
And after you split the instructions, you can split this into a function of its' own: ``ChooseMenu``